src/js/binaryen.js-post.js

```diff
@@ -171,10 +171,10 @@
       });
     };
 
-    this['block'] = function(name, children) {
+    this['block'] = function(name, children, type) {
```

Comment:
for backwards compatibility, we could in theory see if `type` is undefined, and set it to `none` otherwise. this should happen anyhow since its coerced to an int, but i think it would be nicer to do so explicitly.